KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -511,6 +522,27 @@ public DeliveryClient(string projectId, string previewApiKey) : this(new Deliver
                 throw new InvalidOperationException("Preview API and secured Delivery API must not be configured at the same time.");
             }
 
+            return _deliveryOptions.EnableResilienceLogic ? await GetRepeatableDeliverResponseAsync(endpointUrl) : await GetSolitaryDeliveryResponseAsync(endpointUrl);
+        }
+
+        private async Task<JObject> GetRepeatableDeliverResponseAsync(string endpointUrl)
+        {
+            var policyResult = await ResiliencePolicyProvider?.Policy?.ExecuteAndCaptureAsync(() =>
+                {
+                    return SendHttpMessage(endpointUrl);
+                }
+            );
+
+            return await GetResponseContent(policyResult?.FinalHandledResult ?? policyResult?.Result);
+        }
+
+        private async Task<JObject> GetSolitaryDeliveryResponseAsync(string endpointUrl)
```

Comment:
solitary...why not just single? or omit the adjective completely? try to use understandable language, simple english.

Reply:
Allright, alright. :) I've ditched these two methods completely (see above).